app/models/pair_request.rb

```diff
@@ -27,8 +27,8 @@ class PairRequest < ApplicationRecord
 
   validates :when,
     presence: true,
-    inclusion: { in: (Date.today..(Date.today + 1.month)) }
-  validates :duration, presence: true, numericality: { greater_than_or_equal_to: 15.minutes }
+    inclusion: { in: (Date.current..(Date.current + 1.month)) }
+  validates :duration, presence: true, numericality: { greater_than_or_equal_to: 5.minutes }
```

Comment:
Me and Gustavo did have a question about changing this `duration` validation over to 5 minutes minimum. Is that really enough time to hold a pair meeting?

I'm willing to be convinced on this. Just curious to get your thoughts behind it.

Reply:
I don't think we need to really limit this. You could theoretically have a 5 min just "introduce yourself" with someone. When we do the front-end side, we might just make it intervals of 15 by default and offer a way for them to customize.

Reply:
Ah okay that makes sense. We do have 15 minute intervals in the form and thought dropping down to 5 would make it annoying to use.